include/clang/StaticAnalyzer/Core/BugReporter/PathDiagnostic.h

```diff
@@ -371,11 +371,14 @@ class PathDiagnosticPiece: public llvm::FoldingSetNode {
   bool LastInMainSourceFile = false;
   
   /// A constant string that can be used to tag the PathDiagnosticPiece,
-  /// typically with the identification of the creator.  The actual pointer
+  /// typically with the identification of the creator. The actual pointer
   /// value is meant to be an identifier; the string itself is useful for
   /// debugging.
   StringRef Tag;
 
+  /// Internal info, that can optionally be dumped.
+  InternalInfoMap InternalInfos;
```

Comment:
Perhaps a more complying naming would be `InternalInfoMapTy` for the type and `IntermalInfoMap` for the variable.